doc/sphinx/language/gallina-extensions.rst

```diff
@@ -836,18 +831,15 @@ Sections create local contexts which can be shared across multiple definitions.
       :name: @ident already exists. (Let)
       :undocumented:
 
-   .. cmdv:: Let @ident {* @binder } {? : @type } := @term
-      :undocumented:
-
-   .. cmdv:: Let Fixpoint @ident @fix_body {* with @fix_body}
+   .. cmd:: Let Fixpoint @fix_definition {* with @fix_definition }
```

Comment:
I suggest moving this up to a shared block with `Let` as in:

```rst
.. cmd:: Let @ident @def_body
         Let Fixpoint @fix_definition {* with @fix_definition }
         Let CoFixpoint @cofix_definition {* with @cofix_definition }
```

Then the description becomes: "These commands behave like `Definition`, `Fixpoint` and `CoFixpoint` except that..."

Reply:
Does it make sense that `Let Fixpoint` and `Let CoFixpoint` allow a `univ_decl` on the `ident` but `Let` does not?

Reply:
See #11534 for a beginning of discussion. I probably won't try to push this further on my own.